airflow/www/templates/airflow/confirm.html

```diff
@@ -21,7 +21,7 @@
 
 {% block content %}
   {{ super() }}
-  <h2>Wait a minute.</h2>
+  <h2>Wait a minute</h2>
```

Comment:
Changes in this file are too objective to me. They may not be wrong (although I don't see how they improve what), but don't fit well with the rest of the PR.

Reply:
This dot just doesn't look nice in the UI. Short sentences or instructions don't need a dot at the end, especially if they are not included in some bigger text wall, but are just a short notification or instruction. I know it's subjective, but I think it's commonly accepted. Sentences like 'Click OK', 'Check your filters', 'Insert value' do not require a full stop at the end. Also, it is perceived as insincere or even rude nowadays. In my opinion, it looks just better in this case.